File: Leetcode/leetcode-1748-SumOfUniqueElements-[array]-[easy]/Solution.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        visited = dict({})
        for num in nums:
            visited.update({ num: visited.get(num, 0) + 1})
        
        sum = 0
        for k, v in visited.items():
            if (v == 1):
                sum += k
        
        return sum

File: Leetcode/leetcode-1748-SumOfUniqueElements-[array]-[easy]/Solution.java
class Solution {
    public int sumOfUnique(int[] nums) {
        int sum = 0;
        HashMap<Integer, Integer> visited = new HashMap<>();
        for (int num : nums) {
            visited.put(num, visited.getOrDefault(num, 0) + 1);
        }

        for (Map.Entry<Integer, Integer> entry : visited.entrySet()) {
            if (entry.getValue() == 1) {
                sum += entry.getKey();
            }
        }

        return sum;
    }
}